sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/specialized/LeaseAsyncClient.java

```diff
@@ -52,6 +52,7 @@
     private final boolean isBlob;
     private final String leaseId;
     private final AzureBlobStorageImpl client;
+    private final String accountName;
 
     LeaseAsyncClient(HttpPipeline pipeline, String url, String leaseId, boolean isBlob) {
```

Comment:
Why not pass the accountName from the LeaseClientBuilder?

Reply:
agree